tasks/offlinePackagingTasks.ts

```diff
@@ -138,18 +138,26 @@ async function acquireRoslyn(
 }
 
 async function installRazor(packageJSON: any, platformInfo: PlatformInformation) {
-    return await installPackageJsonDependency('Razor', packageJSON, platformInfo);
+    if (!(await installPackageJsonDependency('Razor', packageJSON, platformInfo))) {
+        // Try downloading platform neutral package instead
+        const platformNeutral = new PlatformInformation('neutral', 'neutral');
```

Comment:
I don't think we should have this fallback.  If the platform doesn't exist that we're packaging the vsix for, that is an error in setup authoring.

Fine with having fallbacks at runtime.  But not in creating the vsix.

Reply:
That's fair - I updated the code to not fallback in this method

Reply:
So when is `platformInfo` undefined here? When someone does a build and pack on a machine architecture we don't officially support? Or are we going to start producing platform agnostic builds of the whole extension?

Reply:
I think eventually once we do start publishing our language agnostic servers, this method would be changed to return undefined: https://github.com/dotnet/vscode-csharp/blob/1a8a3a9cc742d4bb13e8a791fffd2b7d4395245b/src/shared/platform.ts#L147

I made the condition undefined based on a similar Roslyn condition: https://github.com/dotnet/vscode-csharp/blob/1a8a3a9cc742d4bb13e8a791fffd2b7d4395245b/tasks/offlinePackagingTasks.ts#L122